doc/cluster-overview.md

```diff
@@ -0,0 +1,15 @@
+# Cluster Overview:
+
+Pravega is a storage system for data streams that has an innovative design and an attractive set of features to cope with today’s Stream processing requirements (e.g., event ordering, scalability, performance, etc.). For deploying Pravega, we have to install the following components as a pre-requisite. Pravega can be deployed in different environments like OpenShift, KubeSpray, GKE, EKS etc
+
```

Comment:
A diagram would be good instead of just text description of components and dependencies.

Reply:
Removed this doc as it is handled in pravega repository https://pravega.io/docs/nightly/admin-guide/cluster-dependencies/

Reply:
Ok.